modules/worker-utils/src/workers/null-worker.ts

```diff
@@ -2,5 +2,5 @@ import {createWorker} from '../lib/worker-api/create-worker';
 
 createWorker(async (data, options) => {
   // @ts-ignore
-  return data;
+  return data && options;
```

Comment:
This is the error.

Reply:
Yes. Remove `options` argument instead. It is not used here.